layouts/index.html

```diff
@@ -17,7 +17,7 @@ <h3 class="posts-item-note" aria-label="Recent Posts">Recent Posts</h3>
             {{ end }}
             {{ if gt (len $posts) 5 }}
             <p>
-                <a href="/posts/">See all posts</a>
+                <a href="{{ .Site.Home.Permalink }}posts/">See all posts</a>
```

Comment:
Nice! I missed this :wink:

Reply:
I think there is ability to get full link from hugo.

In this example we getting first element `.Site.Params.mainSections` and geting matching sectiob object from `.Site.Sections`. This will allow our user to use any name for `posts` directory`
```html
{{ range $firstSection := (where .Site.Sections "Type" "in" (first 1 (.Site.Params.mainSections))) }}
<a href="{{ $firstSection.Permalink }}">See all posts</a>
{{ end }}
```

Reply:
I think that's grate! I will try that on my local project :)